backend/app.js

```diff
@@ -11,6 +11,7 @@ require('dotenv').config()
 
 const app = express()
 app.use(cors())
+app.listen(3001)
```

Comment:
Oops, can't leave this here. Otherwise, the app will crash since it has already started on port `3001`!